README.md

```diff
@@ -1,3 +1,3 @@
 # eslint-config-mapsindoors
 
-ESLint [shareable config](http://eslint.org/docs/developer-guide/shareable-configs.html) used in MapsIndoors JavaScript projects.
+ESLint [configuration](http://eslint.org/docs/developer-guide/shareable-configs.html) used within MapsPeople A/S
```

Comment:
Should this be reverted back into something about MapsIndoors?

Reply:
Lets do that the day we become MapsIndoors

Reply:
Ok.